glide.lock

```diff
@@ -74,7 +74,7 @@ imports:
 - name: github.com/willf/bitset
   version: 2e6e8094ef4745224150c88c16191c7dceaad16f
 - name: golang.org/x/net
-  version: 60c41d1de8da134c05b7b40154a9a82bf5b7edb9
```

Comment:
Can you revert changes to glide.lock file?  Or just checkout out to the copy of glide.lock in master?  This is an annoying side affect of us not pinning `golang.org/x/net` that it is non-deterministic about the version it gets.  

We'll address this in a separate PR.